net-misc/trackma/trackma-0.8.2.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_5,3_6} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Open multi-site list manager for media tracking sites"
+HOMEPAGE="https://github.com/z411/trackma"
+SRC_URI="https://github.com/z411/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+ncurses cli gtk inotify qt5"
+REQUIRED_USE="|| ( cli gtk ncurses qt5 )"
+
+DEPEND="ncurses? ( dev-python/urwid[${PYTHON_USEDEP}] )
```

Comment:
Also here sort by cat/pn, then by USE conditionals.

Reply:
what do you mean by "pn"?
the soft has three different ui and so it need at least one of those

Reply:
cat=category, pn=package name. In other words, the order should be: first unconditional dependencies, sorted by full package name, then all conditional dependencies, sorted by the flag name, and inside those groups again same rules.

```
dev-libs/bar
dev-libs/foo
sys-libs/baz
bar? ( app-misc/bar
    sys-misc/aaaa )
foo? ( app-misc/foo )
```